### PR TITLE
Refactor/fix UI

### PR DESCRIPTION
**Pull request by david-buderus**
Added user comfort.
Fixed different alignment bugs.
Removed old code.

### CODE REVIEW

File: src/main/resources/language/Messages_en.properties [hunk 1]
```diff
@@ -1,10 +1,30 @@
-language = Langauge
+language = Language
+language.table = Language for table import
 excelFile = Excel file
+accessFile = Access file
```
Comment (bastihav) [inline src/main/resources/language/Messages_en.properties]:
```suggestion
accessFile = Access DB file
```

File: src/main/resources/language/Messages_en.properties [hunk 2]
```diff
@@ -37,6 +59,7 @@ battle.info.heal = Heal
 battle.info.attacks.verb = attacks
 battle.info.attacks.ending = 
 battle.info.attack = Attack
+battle.info.selectionHint = Select another person with STRG to show additional options.
```
Comment (bastihav) [inline src/main/resources/language/Messages_en.properties]:
```suggestion
battle.info.selectionHint = Select another entity with STRG to show additional options.
```

File: src/main/resources/language/Messages_en.properties [hunk 3]
```diff
@@ -109,15 +134,178 @@ character.spell.headline.castTime = Cast Time
...
+search.loot.default.container = Container
+search.loot.default.place = Place
+search.loot.button.loot = Loot
+
+search.spell.title = Spell
+search.spell.default.cost = Cost
+
+helper.title = Helper
+
+helper.calender.title = Calender
```
Comment (bastihav) [inline src/main/resources/language/Messages_en.properties]:
```suggestion
helper.calender.title = Calendar
```

File: src/main/resources/language/Messages_en.properties [hunk 4]
```diff
@@ -109,15 +134,178 @@ character.spell.headline.castTime = Cast Time
...
+helper.calender.button.next = Next Day
+helper.calender.button.previous = Previous Day
+helper.calender.isInNewMoon = is in the new moon
+helper.calender.isInFullMoon = is in the full moon
+helper.calender.isInXDaysFullMoon = is in
+helper.calender.isInXDaysFullMoon.ending = days in the full moon
+helper.calender.isInXDaysNewMoon = is in
+helper.calender.isInXDaysNewMoon.ending = days in the new moon
```
Comment (bastihav) [inline src/main/resources/language/Messages_en.properties]: unlocalized name is wrong (calendAr)!

**Review by bastihav — APPROVED** [review]

File: src/main/resources/language/Messages_en.properties (hunk 4 above)
Comment (david-buderus) [reply src/main/resources/language/Messages_en.properties]: Fixed

**Pull request merged**